wire-runtime/src/commonMain/kotlin/com/squareup/wire/internal/Util.kt

```diff
@@ -13,8 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:JvmName("RuntimeUtils")
+
 package com.squareup.wire.internal
 
+import kotlin.jvm.JvmName
```

Comment:
why wasn't this necessary in the other file edit?

Reply:
I guess that's because it's common code, probably `kotlin-stdlib-jvm` has this file in `kotlin.*`, so it doesn't need to be imported, whereas `kotlin-stdlib-common` has it in `kotlin.jvm.*`, hence requires an import.